src/pathNotEq.js

```diff
@@ -0,0 +1,29 @@
+import { pathEq, complement } from 'ramda';
+
+
+/**
+ * Determines whether a nested path on an object has not a specific value,
+ * in R.equals terms. Most likely used to filter a list.
+ *
+ * @func pathNotEq
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.3.0|v2.3.0}
+ * @category Relation
+ * @sig  [k] → a → {a} → Boolean
```

Comment:
```js
@sig [Idx] => a => {a} => Boolean
@sig Idx = String | Int

Reply:
I've thought we can't do that

Reply:
We already can. I have added support some time ago https://github.com/char0n/docdash/commit/bf4b4eebfdaf042c0ea9038419bc26813f243c94